### DISCUSSION TITLE
Thermal model

### DISCUSSION THREAD

**Discussion by Ma2922** [pybamm-team/PyBaMM#2463] [Q&A] [answered]
Hello,Regarding the thermal model, I have the following three questions:

First,Why dimensioned and dimensionless are two different results?
"Negative current collector temperature": T_cn, 
 "Negative current collector temperature [K]": param.Delta_T * T_cn   + param.T_ref,
if Negative current collector temperature = 300K,reference temperature = 298.15K,
so T_cn=300,the dimensioned formula is calculated as 1[K]*300+298.15[K]=598.15[K].

Second,How is the following  temperature formula derived?and is there a reference?
lump.py:
self.rhs = {
            T_vol_av: (
                self.param.B * Q_vol_av + total_cooling_coefficient * (T_vol_av - T_amb)
            )
            / (self.param.C_th * self.param.rho(T_vol_av))
        }

Third,Is the lump model three-dimensional or two-dimensional?

**Comment by brosaplanella** [comment, marked as answer]
1. The [dimensionless](https://en.wikipedia.org/wiki/Nondimensionalization) variables and the dimensional ones have different values by definition. In your case, I suspect that you are taking `T_cn` to be dimensional, as its dimensionless value is expected to be close to 1.
2. Appendix B1 of [this article](https://www.sciencedirect.com/science/article/pii/S0013468621008148) has some details on the derivation of the lumped model.
3. Lumped model is zero dimensional as it takes the temperature to be homogeneous across the battery. The actual dimensions of the battery only come in through the parameters.
